book/src/frost.md

```diff
@@ -0,0 +1,93 @@
+# Understanding FROST
+
+This explain the main concepts and flows of FROST in a generic manner. These
+are important to understand how to use the library, but rest assured that the
+[Tutorial](tutorial.md) will have more concrete information.
+
+FROST is a threshold signature scheme. It allows splitting a Schnorr private key
+into `n` shares for a threshold `t`, such that `t` (or more) participants can
+together generate a signature that can be validated by the matching public key.
+One important aspect is that the resulting signature is indistinguishable from a
+non-threshold signature from the point of view of signature verifiers.
+
+```admonish note
+FROST only supports Schnorr signatures. Therefore it can't produce
+ECDSA signatures.
+```
+
+## Key Generation
+
+There are two options for generating FROST key shares. In both cases, after the
+key generation procedure, each participant will get:
+
+- a **secret share**;
+- a **verifying share** (which can be used by other participants to verify the
+  signature shares they produce);
+- a **group public key**, which is the public key matching the private key that was
+  split into shares.
```

Comment:
We might want to use more words (vs different words) to clarify that verifying signature shares is different than verifying a final signature, or something

Reply:
Done in 5d55082